CHANGELOG.md

```diff
@@ -16,7 +16,30 @@ Please follow the recommendations outlined at [keepachangelog.com](http://keepac
 
 ## Versions
 ### [Unreleased]
-Changes since the last non-beta release.
+#### Added
+- 🚀 **Introducing React Server Components Support!** 🎉
+  - Experience the future of React with full RSC integration
+  - Seamlessly use React Server Components in your Rails apps
+  - Reduce client bundle sizes
+  - Enable powerful new patterns for data fetching
+  - ⚡️ Requires React on Rails Pro - [See the full tutorial](https://www.shakacode.com/react-on-rails-pro/docs/react-server-components-tutorial)
```

Comment:
link does not work

Reply:
The link doesn't work yet, because the docs for RSC are in a PR that hasn't been merged yet.